modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -1209,6 +1209,51 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViweOnlyCorrectAllowedQuantitiesAreSelectableInProductCard {
+		property portal.acceptance = "true";
+
+		task ("Given allowed quantities (1,2,6,8), maximum quantity (7), minimum quantity (2) and multiple quantity (2) are configured for a product") {
+			CommerceJSONProductsAPI._patchCommerceProductConfiguration(
+				allowedOrderQuantities = "1,2,6,8",
+				maxOrderQuantity = 7,
+				minOrderQuantity = 2,
+				multipleOrderQuantity = 2,
+				productName = "U-Joint");
+		}
+
+		task ("Add a account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Commerce",
+				accountEntryType = "Business");
+		}
+
+		task ("When the user checks the quantity options") {
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			Click(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD");
+
+			for (var number : list "2,6") {
+				AssertElementPresent(
+					key_optionValue = ${number},
+					key_productName = "U-Joint",
+					locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD_OPTION_VALUE");
+			}
+		}
+
+		task ("Then only the quantities that meet the configuration are displayed") {
```

Comment:
Then only the quantities that meet the configuration are displayed on the product card